codes/plotting/figA2.py

```diff
@@ -21,16 +22,17 @@ def main():
 
     # validate with JHU
     url = (
-        'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/'
-        'csse_covid_19_time_series/time_series_19-covid-Confirmed.csv')
+        'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/'
```

Comment:
We already have the data downloaded from @kendonB's script right? Is there a reason we don't just pull from that csv?

Reply:
I wanted to compare against the raw data (Kenny implemented some additional cleaning steps, which is great for analysis) but this exercise is a bit different